Sources/App/Extensions/EnvironmentProperties.swift

```diff
@@ -19,6 +19,13 @@ extension Environment {
         return try Environment.get("GITHUB_REPOSITORY")
     }
 
+    static func requiresAllGitHubStatusChecks() throws -> Bool {
+        guard let stringValue: String = try? Environment.get("REQUIRES_ALL_STATUS_CHECKS") else {
+            return false // defaults to only consider required checks
+        }
+        return ["yes", "1", "true"].contains(stringValue.lowercased())
```

Comment:
doesn't `Bool(init: StringLiteral)` do it already?

Reply:
Does that exist? TIL
Though:
1. We're not supposed to use `init(stringLiteral:)` initialisers ourselves nor with non-literal values, those are supposed to be used by compiler only officially
2. I didn't find any `Bool.init(stringLiteral: )`  in the documentation?
3. There's `Bool.init?(_: String)` but the doc for this one says:

> If the description value is any string other than "true" or "false", the result is nil. This initializer is case sensitive.

Which seems a bit too strict to me, especially since by convention most ENV vars that expect a bool value in most contexts expect either `YES` or `1` most of the time, not lowercase `true`